src/exchange/swap/providersPriority.js

```diff
@@ -0,0 +1,3 @@
+const providersPriorityMap = { changelly: 1 };
```

Comment:
Wanted to move this map to a JSON file but apparently *.json files aren't included in the post-build bundle. Tell me if I missed something, will be happy to move this data to a better format file.

Plus we could imagine moving these priorities from here to the CMS, It's a simple idea.

Reply:
I don't think we need priority but rather just rely on the rates themselves, in any case, this mapping is fine by me in case we want to prioritize one over another